account_tax_rounding_method/models/account_move.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from contextlib import contextmanager
+
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        partner_ids = {
+            vals.get("partner_id") for vals in vals_list if vals.get("partner_id")
+        }
+        if len(partner_ids) == 1:
+            partner_id = list(partner_ids)[0]
+            partner = self.env["res.partner"].browse(partner_id)
+            if partner.tax_rounding_method:
+                self = self.with_context(
+                    tax_rounding_method=partner.tax_rounding_method
+                )
+        return super().create(vals_list)
+
+    @contextmanager
+    def _sync_dynamic_line(
+        self,
+        existing_key_fname,
+        needed_vals_fname,
+        needed_dirty_fname,
+        line_type,
+        container,
+    ):
+        if line_type == "tax":
+            tax_rounding_method = (
+                self.env.context.get("tax_rounding_method")
```

Comment:
When we have the `tax_rounding_method` context already, should we still try to get the value from the partner?